indigo_app/templates/indigo_api/document/_content.html

```diff
@@ -26,6 +26,6 @@
   </div>
 </div>
 
-<div class="document-xml-editor">
+<div class="document-xms-editor">
```

Comment:
oops :)

Reply:
please check the editor view to ensure functionality still works

Reply:
Fixed. I've checked and everything seems to be working properly